CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessPane.java

```diff
@@ -371,4 +385,21 @@ protected boolean determineExecuteButtonDisableState(final boolean canExecuteTab
         // or the selected plugins contain invalid parameter values.
         return !queryIsRunning && !canExecuteTabPane;
     }
+    
+    private KeyCombination createCombo(KeyEvent event) {
+        var modifiers = new ArrayList<KeyCombination.Modifier>();
```

Comment:
Can this be declared with a specific type (maybe a `List<KeyCombination.Modifier>` rather than `var`? You could then also remove the explicit type in the instantiation (i.e. `new ArrayList<>();`)